saw-core/prelude/Prelude.sawcore

```diff
@@ -2989,10 +2928,88 @@ emptyFunStack = Nil1 (List1 LetRecType);
 pushFunStack : List1 LetRecType -> FunStack -> FunStack;
 pushFunStack frame stack = Cons1 (List1 LetRecType) frame stack;
 
+-- The type of FunStackE
```

Comment:
Maybe add a little more documentation about what a `FunStackE` is, by describing that it's either an error, an `E`, or a `FrameCall`